packages/test-runner-core/package.json

```diff
@@ -0,0 +1,36 @@
+{
+  "name": "@web/test-runner-core",
+  "version": "0.0.1",
```

Comment:
you should probably start this as 0.0.0 and let changesets handle releasing 0.0.1 properly. If you add a changeset to this PR with a patch, this should happen, and you will have 0.0.1 initial release added to your changelog too.

Reply:
ok